### ISSUE TITLE
Look into badge for our repo from OpenSSF

### ISSUE THREAD

**Issue by LeandraH** [digitalfabrik/integreat-app#2933]
**Is your feature request related to a problem? Please describe.**
As Sven pointed out in https://chat.tuerantuer.org/digitalfabrik/pl/n93dgppw5jgojqbf4r1kit1rcw , there is a badge from Open Source Security Foundation about which best practices a repo follows. Maybe we want to implement that as well in our repo, to encourage us to implement / stick to best practices, and to show people who are interested in contributing that we care about security.

Here is the report for the CMS: https://securityscorecards.dev/viewer/?uri=github.com/digitalfabrik/integreat-cms

And a link to the best practices from the OpenSSF: https://www.bestpractices.dev/de

**Describe the solution you'd like**
Have a shiny new badge in our repo that says that we do a good job on security.

**Describe alternatives you've considered**
We could check on whether we follow the best practices without adding a badge. Or we could ignore this altogether.

**Comment by bahaaTuffaha** [comment]
Should it be added to CMS Repo instead?

**Comment by steffenkleinle** [comment]
> Should it be added to CMS Repo instead?

Nope, this is not related to the CMS repo. Basically the task is to investigate whether we follow the best practices from OpenSSF (in the integreat-app repo) and do the report/add the label if we do. Changed this to research :)

**Comment by bahaaTuffaha** [comment]
hmm I think we can use the cli version to check the score 🤔 
https://scorecard.dev/

**Comment by bahaaTuffaha** [comment]
Adding a workflow looks better than a cli and GitHub token 🤔
